kolibri/plugins/learn/assets/src/state/mutations.js

```diff
@@ -0,0 +1,31 @@
+const Vue = require('vue');
+const coreStore = require('kolibri.coreVue.vuex.store');
+
+const learnMutations = {
+  SET_PAGE_NAME(state, name) {
+    state.pageName = name;
+  },
+  SET_PAGE_STATE(state, pageState) {
+    state.pageState = pageState;
+  },
+  SET_EXAM_LOG(state, examLog) {
+    state.examLog = examLog;
+  },
+  SET_EXAM_ATTEMPT_LOGS(state, examAttemptLogs) {
+    Object.keys(examAttemptLogs).forEach(contentId => {
+      if (!state.examAttemptLogs[contentId]) {
+        Vue.set(state.examAttemptLogs, contentId, {});
+      }
+      Vue.set(state.examAttemptLogs, contentId,
```

Comment:
I think this is a pretty safe refactor since `store.js` is still exporting the same thing. I could undo this an add a parallel implementation of `store.js` inside of `makeStore` if you like.

Without tests it's hard to know whether a refactor is un/safe, though. So I think there is value in doing this to enable more tests going forward.